src/components/Form.tsx

```diff
@@ -0,0 +1,71 @@
+import { Label } from './label-form';
+import { FiSave } from 'react-icons/fi';
+import RadioGroupInputs from './group-radios-inputs';
```

Comment:
Fala Victor, sugiro que altere a ordem de importação para o seguinte modelo:

- Importações sem atribuição (`import 'module';`);
- Importações default (`import module from 'module';`);
- Importações com desestruturação (`import { method } from 'module';`);
- Linha em branco entre importações de módulos públicos e locais;
- Mesma ordem para importações locais.

```suggestion
import { FiSave } from 'react-icons/fi';

import RadioGroupInputs from './group-radios-inputs';
import { Label } from './label-form';
```